app/services/users.js

```diff
@@ -26,3 +28,94 @@ exports.existUserMail = async mail => {
     return new Error(`signUpUser Error => ${error.message}`);
   }
 };
+
+exports.getUserByEmailAndPassword = async (mail, newPassword) => {
+  try {
+    let hasCorrectParameters = false;
+    const userData = await userModel.findOne({ where: { mail } });
+
+    if (objIsNotEmpty(userData) && userData.password) {
+      hasCorrectParameters = await checkPassword(newPassword, userData.password);
+    }
+
+    return hasCorrectParameters ? userData : {};
+  } catch (error) {
+    logger.error(`getUserByEmailAndPassword Error => ${error}`);
+    return new Error(`getUserByEmailAndPassword Error => ${error.message}`);
+  }
+};
+
+exports.getUserByAttribute = async condition => {
+  try {
+    const user = await userModel.findOne({ where: { ...condition } });
+
+    return user;
+  } catch (error) {
+    logger.error(`getUserByAttribute Error => ${error}`);
+    return new Error(`getUserByAttribute Error => ${error.message}`);
+  }
+};
+
+exports.getUserById = id => {
+  try {
+    return userModel.findByPk(id);
+  } catch (error) {
+    logger.error(`getUserById Error => ${error}`);
+    return new Error(`getUserById Error => ${error.message}`);
+  }
+};
+
+exports.getUsersList = async (options = {}) => {
+  try {
+    const returnOptions = {};
+    const conditions = {
+      ...options,
+      attributes: { exclude: ['password'] }
+    };
+
+    const users = await userModel.findAll(conditions);
+    const totalUsers = await userModel.count();
+
+    if (options.limit && options.offset) {
+      returnOptions.limit = options.limit;
+      returnOptions.offset = options.offset;
+    }
+
+    return {
+      users,
+      count: users.length,
+      totalUsers,
+      ...returnOptions
+    };
+  } catch (error) {
+    logger.error(`getUsersList Error => ${error}`);
+    return new Error(`getUsersList Error => ${error.message}`);
+  }
+};
+
+exports.adminSignUp = async data => {
```

Comment:
I think that service is a function that it to do a unique operation. It the name of service is getUserByEmailAndPassword I would wait something like this userModel.findOne({ where: { mail } }).catch(dataBaseError}) and the other actions I would create them in a interactor. What do you think ?